### ISSUE TITLE
Bug: REPL opens in the current buffer

### ISSUE THREAD

**Issue by fstamour** [kovisoft/slimv#67]
Hi, I'm trying to use slimv but the REPL always opens in the same buffer when I connect to a slime server.

I'm currently on windows, vim version:
```
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled May  1 2018 22:10:45)
MS-Windows 32-bit GUI version with OLE support
```
python version 3.6.5

I tried disabling all the other plugins and it still happens.

![image](https://user-images.githubusercontent.com/2881922/39556780-ddf25fb0-4e50-11e8-9827-d171fc290366.png)

**Comment by kovisoft** [comment]
Do you have any slimv variables set in your .vimrc (_vimrc on Windows) file?
What is the buffer name BEFORE opening REPL? What is the output of `:ls` before and after opening the REPL?
Do you have a split screen before opening REPL?
What happens if (without opening the REPL) you execute this command:
`:call SlimvOpenBuffer('REPL')`

We can also debug the buffer opening process this way:
```
:set verbosefile=test.log
:set verbose=20
:call SlimvOpenBuffer('REPL')
```
Could you please send me the resulting `test.log` file?
Thank you for the assistance.

**Comment by fstamour** [comment]
> Do you have any slimv variables set in your .vimrc
no

>What is the buffer name BEFORE opening REPL?

`fuzz2.lisp`

>What is the output of :ls before and after opening the REPL?
Before
![image](https://user-images.githubusercontent.com/2881922/39633095-f7cc8b9c-4f84-11e8-9033-16e1716853a8.png)
After
![image](https://user-images.githubusercontent.com/2881922/39633117-06e4c310-4f85-11e8-9b3f-b90b3e7b9f58.png)

>Do you have a split screen before opening REPL?
no

> What happens if (without opening the REPL) you execute this command:
:call SlimvOpenBuffer('REPL')

A ha! **That's it**

![image](https://user-images.githubusercontent.com/2881922/39633207-5379ad9e-4f85-11e8-8070-a786e5cb611b.png)

Next to this file "fuzz2.lisp", there a repository called `repl`. I tried using slimv on an actual project file that doesn't have a file or folder called `repl` next to it and it worked fine.

And here's the log file: [slimv-67.log](https://github.com/kovisoft/slimv/files/1974783/slimv-67.log)

**Comment by kovisoft** [comment]
In this case you can define a custom name for the REPL buffer.
Add a similar line to your .vimrc (_vimrc) file, using a name of your choice, something that does not collide with any of file/folder names:
```
let g:slimv_repl_name = 'Slimv-REPL'
```

**Comment by fstamour** [comment]
do you plan on putting a fix for that case in slimv?

**Comment by kovisoft** [comment]
Yes, I plan to fix it, but didn't yet figure out how to do it. Probably dynamically changing the REPL buffer name in case a collision found with a file or directory name.

**Comment by kovisoft** [comment]
Fixed in commit https://github.com/kovisoft/slimv/commit/f781a7622ebccd07377600150c79676f76d759f9

**Comment by fstamour** [comment]
I'll try it when I have a few minutes, but I'm sure it's going to be fine.

**Comment by fstamour** [comment]
oups, I might have forgotten about this issue 😂
